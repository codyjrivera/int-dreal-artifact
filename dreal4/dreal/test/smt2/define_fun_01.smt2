(set-logic QF_NRA)
(set-option :produce-models true)
(declare-const z Real)
(define-fun plus_one ((y Real)) Real (+ y 1))
(assert (= (plus_one 4) z))
(check-sat)
(get-model)
