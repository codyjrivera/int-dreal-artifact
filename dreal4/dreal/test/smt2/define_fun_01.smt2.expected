delta-sat with delta = 0.001
z : [5, 5]
(model
  (define-fun z () Real 5)
)
